request.go

```diff
@@ -356,8 +350,12 @@ func unmarshalNode(data *Node, model reflect.Value, included *map[string]*Node)
 				} else if v.Kind() == reflect.Int {
 					at = v.Int()
 				} else {
-					er = ErrInvalidTime
-					break
+					// Return error immediately to ensure a runtime panic doesn't swallow it.
+					return &ErrorObject{
```

Comment:
Hmm ... so, @shwoodard do you mean: how can we support the use case where this specific `invalidTypeErrorTitle` & `invalidTypeErrorDetail` constants can be internationalized?

Maybe I don't understand exactly what you are asking. To be sure though, a user could of course just take the `error` interface returned here (having a concrete type of `*jsonapi.ErrorObject`), and create a new one with internationalized text ... 

Else, it is simply a matter of introducing an i18n system to this package and having users configure it at compile or runtime. We could have it default to English (obvs). I18n and l10n seems like something we definitely would want to tackle after this PR has been merged though, IMHO `;p`.